test/e2e/case5_policy_automation_test.go

```diff
@@ -135,7 +136,448 @@ var _ = Describe("Test policy automation", func() {
 				context.TODO(), "create-service-now-ticket", metav1.GetOptions{},
 			)
 			Expect(err).To(BeNil())
-			Expect(policyAutomation.Object["spec"].(map[string]interface{})["mode"]).To(Equal("disabled"))
+			Expect(
+				policyAutomation.Object["spec"].(map[string]interface{})["mode"],
+			).To(Equal(string(policyv1beta1.Disabled)))
+		})
+		// Create two events then two ansiblejobs for each event
+		It("Test mode = everyEvent without delayAfterRunSeconds", func() {
```

Comment:
Could you make each test clean up the PolicyAutomation object and the AnsibleJob objects so that each test does not depend on the other. This also allows running each test individually for debugging locally.